src/main/java/com/example/music/backend/verification/controller/RegistrationController.java

```diff
@@ -34,7 +34,7 @@ public RegistrationController(UserService userService, MessageSource messages, V
 
         VerificationToken verificationToken = tokenService.getVerificationToken(token);
 
-        if(verificationToken == null) {
+        if (verificationToken.equals(null)) {
```

Comment:
if "verificationToken" is null, NPE would be thrown

Reply:
read about equlas(), returned it as it was